include/aikido/io/trajectory.hpp

```diff
@@ -0,0 +1,33 @@
+#ifndef AIKIDO_IO_TRAJECTORY_HPP_
+#define AIKIDO_IO_TRAJECTORY_HPP_
+
+#include "aikido/trajectory/Spline.hpp"
+#include "aikido/statespace/dart/MetaSkeletonStateSpace.hpp"
+
```

Comment:
Maybe this would be a good place to document the YAML format that we're serializing to and that we're expecting when deserializing.